app/views/system_admin/_tab_nav.html.erb

```diff
@@ -7,4 +7,5 @@
   { name: "Uploads", url: uploads_path },
   { name: "Dead Jobs", url: dead_jobs_path },
   { name: "Pending TRN", url: pending_trns_path },
+  { name: "Pending Awards", url: pending_awards_path },
```

Comment:
Sadly this causes the tab bar to wrap onto a second line. Suggestions on how to resolve this welcome!